lib/jormungandr/test/integration/Main.hs

```diff
@@ -117,7 +118,9 @@ main = hspec $ do
         describe "PR_DISABLED Addresses API endpoint tests" Addresses.spec
         describe "PR_DISABLED Transactions API endpoint tests" Transactions.spec
         describe "Transactions API endpoint tests (Jormungandr specific)"
-            (TransactionsJormungandr.spec @t)
+            (TransactionsAPIJormungandr.spec @t)
+        describe "Transactions CLI endpoint tests (Jormungandr specific)"
+            (TransactionsCLIJormungandr.spec @t)
```

Comment:
Perhaps `TransactionsApiJormungandr` and `TransactionsCliJormungandr` are more readable than `TransactionsAPIJormungandr` and `TransactionsCLIJormungandr`.

(Abbreviation in the middle of a compound name.)

Admittedly, we don't have a standard for this.

Reply:
changed to `TransactionsApiJormungandr` and `TransactionsCliJormungandr`